pkgs/applications/audio/tidal-hifi/default.nix

```diff
@@ -105,6 +105,7 @@ stdenv.mkDerivation (finalAttrs: {
   postFixup = ''
     makeWrapper $out/opt/tidal-hifi/tidal-hifi $out/bin/tidal-hifi \
       --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath finalAttrs.buildInputs}" \
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
```

Comment:
I am not familiar with these vars. It seems they are just ignored if wayland isn't used?

Reply:
https://github.com/NixOS/nixpkgs/pull/147557/files#diff-529f06001d6ab670069fcd059f49af31bb24ae715dc6eeac3149f5f5e0d939c7R548

Reply:
so it's ignored? :p  

Anyway - it seems to work fine in X.